### PR TITLE
Update TESTING.md

### PR DESCRIPTION
**Pull request by Taikelenn**
# Description of the PR
TESTING.md will now include testing of Unicode characters. It seems that this has been an often encountered problem, so testing for that before release should lay waste to this issue once and for all.

# Testing
<table>
  <tr>
    <th>unit</th>
    <th>integration</th>
    <th>manual</th>
  </tr>
  <tr>
    <td>
      <ul>
        <li>- [ ] new <b>unit</b> tests created</li>
        <li>- [X] all <b>unit</b> tests pass</li>
      </ul>
    </td>
    <td>
      <ul>
        <li>- [ ] new <b>integration</b> tests created</li>
        <li>- [X] all <b>integration</b> tests pass</li>
      </ul>
    </td>
    <td>
      <ul>
        <li>- [X] <b>manual</b> testing went well</li>
      </ul>
    </td>
  </tr>
</table>  
  
# Have you updated the [TESTING.md](https://github.com/Aalto-LeTech/aplus-courses/blob/master/TESTING.md) or other relevant documentation on _your_ branch?

- [X] Yes.
- [ ] Not yet. I will do it next.
- [ ] Not relevant.

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -326,19 +326,27 @@ Terminate**.
 
 **6.43** ASSERTION: A Scala REPL for **O1Library** opens.
 
-**6.44** Close the REPL.
+**6.44** Execute the following command in the REPL: `println("Myöhässä")`
+
+**6.45** ASSERTION: The word appears correctly, without any garbled letters.
+
+**6.46** Execute the following command in the REPL: `println("世界です")`
```
Comment (OlliKiljunen) [inline TESTING.md:333]: Combining these strings into a single input would make it a bit faster to go through.

**Review by OlliKiljunen — APPROVED** [review]
Our plan, at some point, was to make the REPL part shorter. This addition is a bit counterproductive to that end. But necessary, as discussed earlier; thus approved.

File: TESTING.md (hunk 1 above)
Comment (Taikelenn) [reply TESTING.md:333]: Good point
